### DISCUSSION TITLE
Questions & Help Requests are now in Discussions!

### DISCUSSION THREAD

**Discussion by yuvaltassa** [google-deepmind/mujoco#1940] [Announcements]
Hi everyone,

To improve the organization and efficiency of our project, we are making a small but important change in how we handle community interactions.

**Going forward:**

**Questions and Help Requests** should be submitted via [GitHub Discussions](https://github.com/google-deepmind/mujoco/discussions/categories/asking-for-help). This is the best place to ask for help, share knowledge by providing answers, and engage with other community members.
  
**GitHub Issues** should be reserved **ONLY** for **bug reports** and **feature requests**. This will help us maintain a clear focus on tracking and resolving actionable items related to the project.

Why the change?  
By utilizing Discussions for questions and support, we can:
- Foster a more collaborative and searchable space for everyone.
- Keep the Issues section clean and focused on development-related tasks.

Thank you for your understanding and cooperation! We’re excited to continue improving the project with your help.

If you have any questions about this change, feel free to start a new discussion!

Best regards,  
The MuJoCo development team

**Comment by 1wang861** [comment]
Hello, I would like to use myosuite for upper limb exoskeleton customization, and I hope you can give me learning suggestions

**Reply by Balint-H** [reply]
Hello! Myosuite is a separate (but of course connected) community, try submitting a discussion at its repo: https://github.com/MyoHub/myosuite
